Can't buy fish bait

I'm... not quite sure why it's crashing. Could it be because of the type category, maybe?
```
2024-10-19T02:09:29:053 [0.1.0] debug:  handling acquision of /Lotus/StoreItems/Types/Game/FishBait/GenericFishBait
2024-10-19T02:09:29:055 [0.1.0] debug:  store category Types
2024-10-19T02:09:29:055 [0.1.0] debug:  type category Game
2024-10-19T02:09:29:056 [0.1.0] error:  unable to add item: /Lotus/Types/Game/FishBait/GenericFishBait
Error: unable to add item: /Lotus/Types/Game/FishBait/GenericFishBait
    at addItem (C:\Games\Warframe - Server Emulator\SpaceNinjaServer\src\services\inventoryService.ts:345:11)
    at handleTypesPurchase (C:\Games\Warframe - Server Emulator\SpaceNinjaServer\src\services\purchaseService.ts:163:33)
    at handleStoreItemAcquisition (C:\Games\Warframe - Server Emulator\SpaceNinjaServer\src\services\purchaseService.ts:97:42)
    at handlePurchase (C:\Games\Warframe - Server Emulator\SpaceNinjaServer\src\services\purchaseService.ts:32:36)
    at purchaseController (C:\Games\Warframe - Server Emulator\SpaceNinjaServer\src\controllers\api\purchaseController.ts:10:42)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
[ERROR] 02:09:29 Error: unable to add item: /Lotus/Types/Game/FishBait/GenericFishBait
```

And in rejecting my request, it's crashing. The server stopped working altogether after that [ERROR] and booted me out of the game because "the server wasn't responsive", which is how I noticed the crash.

--- a/src/services/inventoryService.ts
+++ b/src/services/inventoryService.ts
@@ -35,6 +35,7 @@ import {
     ExportBoosterPacks,
     ExportCustoms,
     ExportFlavour,
+    ExportGear,
     ExportRecipes,
     ExportResources,
     ExportUpgrades
@@ -186,6 +187,22 @@ export const addItem = async (
             }
         };
     }
+    if (typeName in ExportGear) {
+        const inventory = await getInventory(accountId);
+        const consumablesChanges = [
+            {
+                ItemType: typeName,
+                ItemCount: quantity
+            } satisfies IConsumable
+        ];
+        addConsumables(inventory, consumablesChanges);
+        await inventory.save();
+        return {
+            InventoryChanges: {
+                Consumables: consumablesChanges
+            }
+        };
+    }
 
     // Path-based duck typing
     switch (typeName.substr(1).split("/")[1]) {
@@ -329,21 +346,6 @@ export const addItem = async (
                         };
                     }
                     break;
-                case "Restoratives": // Codex Scanner, Remote Observer, Starburst
-                    const inventory = await getInventory(accountId);
-                    const consumablesChanges = [
-                        {
-                            ItemType: typeName,
-                            ItemCount: quantity
-                        } satisfies IConsumable
-                    ];
-                    addConsumables(inventory, consumablesChanges);
-                    await inventory.save();
-                    return {
-                        InventoryChanges: {
-                            Consumables: consumablesChanges
-                        }
-                    };
             }
             break;
     }

